README.md

```diff
@@ -1,8 +1,16 @@
-# formatex
```

Comment:
Why have you changed the readme from Markdown to raw HTML?

Reply:
I wanted to center the content so it nicely aligns together. We can do without it though. Should I revert that back to how it was?

Reply:
Yeah that would be best, thanks.

Or, open a separate PR/issue with readme-specific changes.

Reply:
Reverted!